wbooks/src/App.js

```diff
@@ -2,22 +2,19 @@ import React from 'react';
 import './App.css';
 import BooksDashboard from './app/screens/BooksDashboard.js';
 import BookDetail from './app/screens/BookDetail/BookDetail.js'
-import Login from './app/screens/Login/Login.js'
+import Login from './app/screens/Login/index.js'
```

Comment:
You don't need to write /index.js 😅 , just `import Login from './app/screens/Login/index.js'`